tests/Feature/QuestionTest.php

```diff
@@ -0,0 +1,106 @@
+<?php
+
+namespace Tests\Feature;
+
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Tests\TestCase;
+
+use App\Church;
+use App\Question;
+
+class QuestionTest extends TestCase
+{
+  use RefreshDatabase;
+
+  protected $response;
+	
+  protected function setUp(): void
+  {
+    parent::setUp();
+
+    $church_response = $this->post(route('churches.store'), [
+      'id' => 1,
```

Comment:
For security, we probably don't want to allow database id to be set. Instead, you can pull id out of the church response and pass it on to the question store.